osprey-mock-service.js

```
@@ -82,16 +96,18 @@ function handler (method) {
   var statusCode = getStatusCode(method)
   var response = (method.responses || {})[statusCode] || {}
   var bodies = response.body || {}
-  var headers = response.headers || method.headers || {}
+  var headers = {}
   var types = Object.keys(bodies)
 
   // Set up the default response headers.
-  Object.keys(headers).forEach(function (key) {
-    var value = headers[key]
-    if (value && value.default) {
-      headers[key] = value.default
-    }
-  })
+  if (Array.isArray(response.headers)) {
```

If we decide to do anything with `method.headers` later on, we may need this. For now though, I would remove that check since we expect the parser to always return `response.headers` as an array. 

This also means that if ever the parser returns anything else, it would throw an error here instead of failing silently (by "failing silently", I mean "not mocking those headers").